ykcompile/src/lib.rs

```diff
@@ -125,8 +129,26 @@ impl TraceCompiler {
         }
     }
 
+    /// Returns the currently assigned register for a given `Local`. Similar to `local_to_reg` but
+    /// read-only, i.e. this function doesn't assign `Local`'s to registers.
+    fn get_reg(&self, local: &Local) -> Result<u8, CompileError> {
```

Comment:
AFAICS this function is dead, so we should (later) squash https://github.com/softdevteam/yk/pull/67/commits/d3394ddde1bfc23dffdbb5bd541fedd56cc36071 into this commit?

Reply:
I see you already mentioned this.

Reply:
Yes, I noticed shortly after pushing, but didn't know if force pushing is okay if I haven't assigned anyone yet.

Reply:
You did the right thing.